index.html

```diff
@@ -1295,14 +1295,15 @@ <h2>Security Considerations</h2>
     </p>
 
     <p class="advisement">
-      Avoid open source implementations that are declared a "defacto
-      standard", but lack open standard technical specifications.
+      Prefer technologies based on open standard technical specifications.
+      Avoid technologies declared <em>de facto</em> standard based on
+      particular implementations but lacking open standards.
     </p>
 
     <p class="advisement">
-      Support for legacy cryptography systems such as
+      Consider support for legacy cryptography systems such as
```

Comment:
```suggestion
      Consider support for widely adopted cryptography systems such as
```

Reply:
This might be controversial, but i think legacy has the wrong connotation here.

Reply:
```suggestion
      Consider support for historically prevalent cryptography systems such as
```
?

Reply:
Added both suggested phrasings, and removed the latter clause which I think is made redundant: c539bca8f218e38ad07538ab6f51eb161e3bb001